src/pages/billing/List.js

```diff
@@ -44,8 +44,8 @@ const billingProjectNameValidator = existing => ({
   presence: { allowEmpty: false },
   length: { minimum: 6, maximum: 30 },
   format: {
-    pattern: /^[a-z]([a-z0-9-])*$/,
```

Comment:
why did we decide no spaces?

Reply:
Yes, and also SAM requires that spaces are encoded, so we would need Rawl's regex to allow for `%` as well.

Reply:
This is exactly what `\w` is, should this be `/\w+/`? As written, it would validate 0-length.

Reply:
Done

Reply:
drive by comment: I like the brevity of it, but I think the readability suffers (especially if you don't know what `\w` means in regex)

Reply:
I think it is probably okay with the \w. My thinking is that \w while not necessarily as clear at a glance, is something that is easily look-up-able if someone needs to futz with the regex for any reason. So the readability for folks is probably fine for when they need to make changes?